### PR TITLE
[fi] Add alert handler result

### PR DESCRIPTION
**Pull request by nasahlpa**
This commit adds the expected response from the alert handler to the FI scripts.

The device code is located in lowRISC/opentitan#21918

### CODE REVIEW

**Review by johannheyszl — APPROVED** [review]
Thanks @nasahlpa

File: ci/cfg/ci_otbn_fi_vcc_dummy_cw310.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ target:
   target_freq: 24000000
   baudrate: 115200
   protocol: "ujson"
-  port: "/dev/ttyACM1"
+  port: "/dev/ttyACM_CW310_1"
```
Comment (vogelpi) [inline ci/cfg/ci_otbn_fi_vcc_dummy_cw310.yaml:11]: This change is probably not intentional, right?

File: fault_injection/configs/pen.global_fi.ibex.char.reg_op_loop.cw310.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ target:
   target_freq: 24000000
   baudrate: 115200
   protocol: "ujson"
-  port: "/dev/ttyACM1"
+  port: "/dev/ttyACM_CW310_1"
```
Comment (vogelpi) [inline fault_injection/configs/pen.global_fi.ibex.char.reg_op_loop.cw310.yaml:11]: same here

File: fault_injection/fi_ibex.py [hunk 3]
```diff
@@ -97,12 +98,28 @@ def fi_parameter_sweep(cfg: dict, target: Target, fi_gear,
 
             # Read response.
             response = ot_communication.read_response()
+            response_compare = response
+            expected_response = cfg["test"]["expected_result"]
+
+            # If the text decides to ignore alerts triggered by the alert
```
Comment (vogelpi) [inline fault_injection/fi_ibex.py:104]: text -> test

File: fault_injection/fi_otbn.py [hunk 4]
```diff
@@ -96,12 +97,28 @@ def fi_parameter_sweep(cfg: dict, target: Target, fi_gear,
 
             # Read response.
             response = ot_communication.read_response()
+            response_compare = response
+            expected_response = cfg["test"]["expected_result"]
+
+            # If the text decides to ignore alerts triggered by the alert
```
Comment (vogelpi) [inline fault_injection/fi_otbn.py:103]: text -> test

**Review by vogelpi — APPROVED** [review]
LGTM, but I think there some changes made it into the yaml which are specific to your experimental setup and that shouldn't be there.

File: fault_injection/configs/pen.global_fi.ibex.char.reg_op_loop.cw310.yaml (hunk 2 above)
Comment (nasahlpa) [reply fault_injection/configs/pen.global_fi.ibex.char.reg_op_loop.cw310.yaml:11]: Oh yes, sorry about that. Fixed.

**Pull request merged**